ToggleButton/ToggleButton.js

```diff
@@ -0,0 +1,212 @@
+/**
+ * Agate styled button components and behaviors.
+ *
+ * @example
+ * <Button small>Hello Enact!</Button>
+ *
+ * @module agate/Button
+ * @exports Button
+ * @exports ButtonBase
+ * @exports ButtonDecorator
```

Comment:
While it may still be too early to have proper documentation (descriptions) for agate components, I'd like to make sure we are providing proper examples and @export docs. Could we update the `Button` references to `ToggleButton` and make sure we're only including what we're exporting here (`ToggleButton`, `ToggleButtonBase`)?

Reply:
fixed!